### DISCUSSION TITLE
Colors don't match for go

### DISCUSSION THREAD

**Discussion by webstradev** [olimorris/onedarkpro.nvim#178] [Color match]
Hey there,

I'm seeing a few discrepancies between onedarkpro in go and in visual studio code.

I've tried overriding various colors but to no avail.

Main difference is that a lot of text which is white(or more specifically #abb2f in visual studio code is red (or more specifically #e06c75) in Neovim.

Neovim:
![image](https://user-images.githubusercontent.com/82543732/230638525-6d655d94-948c-45d5-9f81-63921a777559.png)

Visual Studio Code:
![image](https://user-images.githubusercontent.com/82543732/230638645-94489964-54b4-457f-a11e-547b062523e8.png)

**Comment by olimorris** [comment]
Oh I say. Big difference. I will sort that

**Reply by webstradev** [reply]
Awesome thanks!

**Reply by olimorris** [reply]
I've just pushed initial support for `Go`. Your use case above currently needs some custom Treesitter queries so will work on that. I've included an example `go.go` file in the `examples` folder. Feel free to open up a PR and add any additional Go syntax to that

**Reply by webstradev** [reply]
Awesome thanks for this, I'll be taking a look soon and ill look out if there is any go syntax I can add.
